articles/quickstart.md

```diff
@@ -162,179 +205,143 @@ A quantum operation transforms the state of a qubit. Sometime people talk about
+```dotnetcli
+Test results (# of 0s, # of 1s):
+(0, 1000)
 ```
 
-The program will exit after you press a key.
+If you try with `initial = Zero` you should observe:
 
-* * *
+```dotnetcli
+dotnet run --count 1000 --initial Zero
+Test results (# of 0s, # of 1s):
+(1000, 0)
+```
 
 ## Prepare superposition
 
-**Overview** Now let’s look at how Q# expresses ways to put qubits in superposition.  Recall that the state of a qubit can be in a superposition of 0 and 1.  We’ll use the `Hadamard` operation to accomplish this. If the qubit is in either of the classical states (where a measurement returns `Zero` always or `One` always), then the `Hadamard` or `H` operation will put the qubit in a state where a measurement of the qubit will return `Zero` 50% of the time and return `One` 50% of the time.  Conceputually, the qubit can be thought of as halfway between the `Zero` and `One`.  Now, when we simulate the `TestBellState` operation, we will see the results will return roughly an equal number of `Zero` and `One` after measurement.  
-
-First we'll just try to flip the qubit (if the qubit is in `Zero` state will flip to `One` and vice versa). This is accomplished by performing an `X` operation before we measure it in `TestBellState`:
+**Overview** Now let’s look at how Q# expresses ways to put qubits in
+superposition.  Recall that the state of a qubit can be in a superposition of 0
+and 1.  We’ll use the `Hadamard` operation to accomplish this. If the qubit is
+in either of the classical states (where a measurement returns `Zero` always or
+`One` always), then the `Hadamard` or `H` operation will put the qubit in a
+state where a measurement of the qubit will return `Zero` 50% of the time and
+return `One` 50% of the time.  Conceputually, the qubit can be thought of as
+halfway between the `Zero` and `One`.  Now, when we simulate the `TestBellState`
+operation, we will see the results will return roughly an equal number of `Zero`
+and `One` after measurement.  
+
+First we'll just try to flip the qubit (if the qubit is in `Zero` state will
+flip to `One` and vice versa). This is accomplished by performing an `X`
+operation before we measure it in `TestBellState`:
 
 ```qsharp
 X(qubit);
 let res = M(qubit);
 ```
 
-Now the results (after pressing `F5`) are reversed:
+Now the results are reversed:
+
+```dotnetcli
+dotnet run --count 1000 --initial One
+Test results (# of 0s, # of 1s):
+(1000, 0)
 
-```Output
-Init:Zero 0s=0    1s=1000
-Init:One  0s=1000 1s=0
+dotnet run --count 1000 --initial Zero
+Test results (# of 0s, # of 1s):
+(0, 1000)
 ```
 
-However, everything we've seen so far is classical. Let's get a quantum result. All we need to do is replace the `X` operation in the previous run with an `H` or Hadamard operation. Instead of flipping the qubit all the way from 0 to 1, we will only flip it halfway. The replaced lines in `TestBellState` now look like:
+However, everything we've seen so far is classical. Let's get a quantum result.
```

Comment:
```suggestion
However, everything we've seen so far has been analogous to classical operations on classical bits. Let's get a quantum result.
```

I'm hesitant about the overuse of "classical" when referring to the subset of quantum that's analogous to quantum. In a lot of the prose it's been fine, like with the comp. basis states as the intro paragraph defined them as being called *classical states*. But I think saying this "is classical" runs a notable risk of confusing newcomers.